dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -68,8 +70,13 @@
     private AuthorizationFeatureService authorizationFeatureService;
 
     @Autowired
-    private ConverterService converterService;
+    private SiteConverter siteConverter;
```

Comment:
This was in the original description.
Permissions are retrieved based on the request. ITs using the ConverterService don't have a request. This implies no permissions can be included, and the information cannot be returned unless it's publicly accessible.
The SiteConverter is deeper inline code, and doesn't use the ConverterService nor the permissions. So it can still be used to create the Rest resource. It would have been better if Converters were never used in the ITs, but refactoring that would fall outside the scope